canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -405,22 +419,52 @@ class ToolTip extends React.Component {
 		if (this.props.className) {
 			tipClass += " " + this.props.className;
 		}
-
+		let linkClicked = false;
 		let link = null;
 		if (this.state.isTooltipVisible && this.props.tooltipLinkHandler && this.props.link) {
 			const linkInformation = this.props.tooltipLinkHandler(this.props.link);
 			// Verify tooltipLinkHandler returns object in correct format
 			if (typeof linkInformation === "object" && linkInformation.label && linkInformation.url) {
-				link = (<Link
-					className="tooltip-link"
-					id={this.props.link.id}
-					href={linkInformation.url}
-					target="_blank"
-					rel="noopener"
-					visited={false}
+				link = (<div
+					ref={(ref) => (this.linkRef = ref)}
+					onKeyDown={(evt) => {
+						evt.stopPropagation();
+						evt.preventDefault();
+
+						if (evt.key === "Escape") { // When Esc is pressed shift the focus to tooltip icon so that user can navigate following elements.
```

Comment:
Please avoid having inline comments like this stretch too far across the screen. Short comments are OK but with one this length I'd prefer it to be above the `if` statement. BTW - I do appreciate this file even has code that stretches a long way across the page so that is not a good model to follow. Ah! If only I could go back and re-review it ;-)